### PR TITLE
Write docs for sugar3.graphics.toolbox

### PR DESCRIPTION
**Pull request by ndefilippis**

### CODE REVIEW

**Comment by samdroid-apps** [comment]
[Unrelated]

Does anybody use this module, grepping says not any more:

```
[I] sam@vogon-constructor-ship ~/sugar-build (master) [I]> grep -r 'graphics.toolbox' --include='*.py'
sugar-toolkit/src/sugar/activity/widgets.py:from sugar.graphics.toolbox import Toolbox
build/out/install/lib/python2.7/site-packages/sugar/activity/widgets.py:from sugar.graphics.toolbox import Toolbox
```

File: src/sugar3/graphics/toolbox.py [hunk 1]
```diff
@@ -15,9 +15,14 @@
...
-"""
-STABLE.
-"""
+'''
+A toolbox arranges a group of toolbars in a verical fashion.
+Toolbars are assigned an index and can be accessed using this index.
+Indicies are generated in the order the toolbars are added.
+
+Emits signal `current-toolbar-changed` signal when the
+current toolbar is changed. This signal takes no arguments.
```
Comment (samdroid-apps) [inline src/sugar3/graphics/toolbox.py]: Lin 68:

`self.emit('current-toolbar-changed', notebook.props.page)`

That means that the first argument is the notebook.props.page.  (whatever that is)

Comment (samdroid-apps) [inline src/sugar3/graphics/toolbox.py]: Also please document the signal in the class section.

File: src/sugar3/graphics/toolbox.py [hunk 2]
```diff
@@ -58,6 +68,17 @@ def _notify_page_cb(self, notebook, pspec):
...
+        Adds a toolbar to this toolbox. Toolbar will be added
+        to the bottom of this toolbox, and it's index will be 
+        1 greater than the previously added index. (index will be
+        0 if it is the first toolbar added)
+        
+        Args:
+            name (string): name of toolbar to be added
+            
+            toolbar (.. class::Gtk.Toolbar): Gtk.Toolbar to be appended to this toolbox
+        '''
```
Comment (samdroid-apps) [inline src/sugar3/graphics/toolbox.py]: syntax should be

    :class:`Gtk.Toolbar`

File: src/sugar3/graphics/toolbox.py [hunk 3]
```diff
@@ -89,9 +116,18 @@ def remove_toolbar(self, index):
             self._separator.hide()
 
     def set_current_toolbar(self, index):
+        '''
+        Sets the current toolbar to that of the index specified
```
Comment (samdroid-apps) [inline src/sugar3/graphics/toolbox.py]: What does this do?  DOes this like hide the other toolbars?

**Comment by quozl** [comment]
@samroid-apps wrote:
> Does anybody use this module

None of the 39 activities in the OLPC Ubuntu Sugar build use this class.  But it's an API, so we get to keep it.  Maybe we can also deprecate it?

**Comment by ndefilippis** [comment]
Changes made

**Comment by quozl** [comment]
Reviewed.

**Pull request merged**